docs.md

```diff
@@ -583,6 +583,26 @@ let signer = uniqueHelper.util.fromSeed('//Alice');
 let success = await collection.setConstOnChainSchema(signer, '{"nested":{"onChainMetaData":{"nested":{"NFTMeta":{"fields":{"ipfsJson":{"id":1,"rule":"required","type":"string"},"gender":{"id":2,"rule":"required","type":"Gender"},"traits":{"id":3,"rule":"repeated","type":"PunkTrait"}}},"Gender":{"options":{"Female":"{\"en\": \"Female\"}","Male":"{\"en\": \"Male\"}"},"values":{"Female":1,"Male":0}},"PunkTrait":{"options":{"SMILE":"{\"en\": \"Smile\"}","SUNGLASSES":"{\"en\": \"Sunglasses\"}","MUSTACHE":"{\"en\": \"Mustache\"}","BALD":"{\"en\": \"Bald\"}"},"values":{"SMILE":0,"SUNGLASSES":1,"MUSTACHE":2,"BALD":3}}}}}}');
 ```
 
+### getCollectionTokenNextSponsored
+
+```typescript
+async getCollectionTokenNextSponsored(collectionId : Number, tokenId : Number, addressObj : {Substrate?: string, Ethereum?: string}): Promise<Number> | null
+```
+
+Get number of blocks when sponsored transaction is available. Returns null if sponsorship hasn't been set.
+
+Example:
+
+```javascript
+const collectionId = (await helper.mintNFTCollection(alice, {name: 't1', description: 't1', tokenPrefix: 'tst'})).collectionId;
+
+const token = (await helper.mintNFTToken(alice, {collectionId, owner: bob.address, variableData: 'bob token', constData: "0x1111"})).token.tokenId;
+
+let res = await helper.setNFTCollectionSponsor(alice, collectionId, '5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY');
+
+await helper.confirmNFTCollectionSponsorship(alice, collectionId);
+let test = await helper.getCollectionTokenNextSponsored(collectionId, token, {Substrate: '5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty'});
```

Comment:
Use `uniqueHelper` instead of helper (See other examples)
You can use 1 as collectionId and 1 as tokenId (code before is not necessary)

But you need to add other example for UniqueNFTCollection:

```javascript
let collection = new UniqueNFTCollection(1, uniqueHelper);
await collection.getTokenNextSponsored(1, {Substrate: '5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY'});
```